src/linker/utilities/singularity_utils.py

```diff
@@ -25,13 +26,11 @@ def _build_container(results_dir: Path, step_dir: Path) -> None:
     _run_cmd(results_dir, cmd)
 
 
-def _run_container(results_dir: Path, step_dir: Path) -> None:
-    cmd = (
-        f"singularity run --pwd {step_dir} "
-        f"--bind {results_dir}:/app/results "
-        f"--bind {step_dir}/input_data:/app/input_data "
-        f"{results_dir}/image.sif"
-    )
+def _run_container(input_data: List[Path], results_dir: Path, step_dir: Path) -> None:
+    cmd = f"singularity run --pwd {step_dir} " f"--bind {results_dir}:/results "
```

Comment:
The formatting of this is weird. I think the two strings need to be combined into one?

Reply:
Whoa - yeah not sure what happened there. I don't even see how this worked tbh